src/main/java/micdoodle8/mods/galacticraft/api/spaceprojects/GCGlobalVariableWorldSavedData.java

```diff
@@ -0,0 +1,106 @@
+package micdoodle8.mods.galacticraft.api.spaceprojects;
+
+import static micdoodle8.mods.galacticraft.api.spaceprojects.GCGlobalVariableStorage.*;
+
+import cpw.mods.fml.common.eventhandler.SubscribeEvent;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.HashMap;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.world.World;
+import net.minecraft.world.WorldSavedData;
+import net.minecraft.world.storage.MapStorage;
+import net.minecraftforge.event.world.WorldEvent;
+
+public class GCGlobalVariableWorldSavedData extends WorldSavedData {
+
+    public static GCGlobalVariableWorldSavedData INSTANCE;
+
+    private static final String DATA_NAME = "GalaxySpace_GlobalVariableWorldSavedData";
+
+    private static final String GlobalSpaceElevatorNameNBTTag = "GalaxySpace_GlobalSpaceElevatorName_MapNBTTag";
```

Comment:
1) constants are all uppercase by convention
2) please don't hardcode any GS stuff into GC, rather put a List here which you populate from GS

Reply:
Oops copy paste error on this one